astrospec/spectrum1d.py

```diff
@@ -8,10 +9,18 @@
 
 from warnings import warn
 
+from .wcs import WCSTable
+
 
 class Spectrum1D(NDSlicingMixin, NDIOMixin, NDData):
-    """A 1 dimensionan spectrum.
-    
+    """A 1-dimensional spectrum.
+
+    Attributes
+    ----------
+
+    Methoods
+    --------
+
```

Comment:
I don't see a reason to list the attributes and methods here. You can always get them with `spectrum1d.<tab>` in IPython, with `dir(spectrum1d)` or in the sphinx docs where we auto-generate a list of all methods (and those attributes that have a sphinx compatible docstring).